.github/workflows/mingw.yaml

```diff
@@ -0,0 +1,35 @@
+# Test a cross-build of the branch on mingw.
+# This test will fail until the mingw build will be fully functional
+# Remove this workflow prior to merging, it belongs to its own PR
```

Comment:
AFAICT, the primary remaining/old issue is the removal of the temporary mingw.yaml workflow. Once that is done, I can dismiss my blocking vote, but you may want to wait until a green light from @yadij before removing that file...

Reply:
It's all right. I can remove it here and restore it in the next PR, nothing will be lost

Reply:
done